README.md

```diff
@@ -723,7 +723,7 @@ If extending Tavor yourself is not for you, but you still need new features, you
 
 The fuzzing filter code and all officially implemented fuzzing filters can be found in the [github.com/zimmski/tavor/fuzz/filter package](/fuzz/filter) and its sub-packages.
 
-A fuzzing filter has to implement the `Filter` interface which is exported by the [github.com/zimmski/tavor/fuzz/filter package](/fuzz/filter). The interface defines the `Apply` method that applies the filter onto a token which is passed to the method. Therefore, the method's concern is only one token at a time. If an error is encountered during the filter execution, the error return argument is not nil. On success a replacement for the token is returned. This can be either `nil`, meaning the token should not be replaced, or a slice of tokens which will replace the old token using an alternation group.
+A fuzzing filter has to implement the `Filter` interface which is exported by the [github.com/zimmski/tavor/fuzz/filter package](/fuzz/filter). The interface defines the `Apply` method that applies the filter onto a token which is passed to the method. Therefore, the method's concern is only one token at a time. If an error is encountered during the filter execution, the error return argument is not nil. On success a replacement for the token is returned. This replacement can be `nil`, meaning the token should not be replaced. A token can be replaced by multiple tokens simply by returning a list token, such as [All](/token/lists/all.go) or [One](/token/lists/one.go).
```

Comment:
The additional sentence with the reference to the All and One token is not needed. Simple state that if the token return argument is not nil that it will replace the original token.